tsdb/index/index.go

```diff
@@ -628,10 +610,10 @@ func (w *Writer) writeLabelIndices() error {
 	values := []uint32{}
 	for d.Err() == nil && cnt > 0 {
 		cnt--
-		d.Uvarint()                           // Keycount.
-		name := d.UvarintBytes()              // Label name.
-		value := yoloString(d.UvarintBytes()) // Label value.
-		d.Uvarint64()                         // Offset.
+		d.Uvarint()               // Keycount.
+		name := d.UvarintBytes()  // Label name.
+		value := d.UvarintBytes() // Label value.
```

Comment:
@aknuds1 asked about this change.  The reason is that we need to allocate a copy when adding the name to the cache (on line 629), but we don't want to allocate if the entry is already there.
This behaviour is what Go does when presented with a `[]byte` converted to `string()` as a map key.

Reply:
Update: actually 629 is only reading the map, so it will never allocate.  
Still, I think it's nicer to remove a use of `yolo`.